extensions/azurecore/src/account-provider/auths/azureAuthError.ts

```diff
@@ -0,0 +1,24 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the Source EULA. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+export class AzureAuthError extends Error {
```

Comment:
This class allows me to know which errors are important enough to highlight to the user.

Reply:
I've always felt that we should have a DisplayableError or something available in core/extension host. We still end up showing a lot of text directly from errors which isn't currently localized...